### PR TITLE
Grammar and typo corrections

### PR DESCRIPTION
**Pull request by HectorAE**
Hi. I think that this is a cool project that I would like to see improve as more people migrate to GNU social, so I wanted to do my part to help. I noticed that there were many typographical errors, grammar mistakes, and the like on the website. I fixed all that I could find on the homepage and the TODO page, and additionally reworded the information where it was stated in a way that was hard to follow. I hope that you'll find these changes satisfactory, and if so, I will be very happy to help in the future doing more of the same across the site. I'm not much of a PHP programmer, so I can't help with the functionality yet, but I think it's important that the vital information on Fediverse.org is presented well so that it's easy to understand and doesn't inadvertently trigger the grammar Nazis (like me) who would otherwise be impressed by the functionality. ;D

### CODE REVIEW

**Comment by tuxxus** [comment]
Thanks a lot for the collaboration HectorAE, and sorry for having an outdate repo here. Shame on me.
I corrected many mistakes on the site directly (sorry), and forgot to update the repo, i will review the changes one by one and marge. I also make sure the repo is up to date in the future.

Being a non-native english speaker, this kind of help is very appreacited. I need to write from scratch many sections of the site, like the F.A.Q page. That would also be a great contribution so I can focus in the code side of things.

Again, thanks a lot for this. I will make sure to merge this changes when they are not already solved in the live version.

Cheers!

**Comment by tuxxus** [comment]
Manually added changes when needed. Cheers.

**Comment by echosa** [comment]
@tuxxus At least one of the changes presented in this PR didn't make it to the live site. The site still says "attemp" instead of "attempt". It seems like there are two instance of this in the repo. See screenshots.

screen shot 2016-04-28 at 10 54 31 pm
screen shot 2016-04-28 at 10 54 08 pm
